src/cf-typography/usage.md

```diff
@@ -42,42 +42,43 @@ project by duplicating the variable `@key: value`.
 
 ### Color variables
 
-`$color-` variables referenced in comments are from 18F's
-[U.S. Web Design Standards](https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
+Color variables referenced in comments are from [cf-core cf-brand-colors.less](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/src/cf-brand-colors.less).
 
 ```
+// Color variables
```

Comment:
TRIVIAL: We don't need this because the heading is `### Color variables` already. It ends up looking like:

![screen shot 2017-09-06 at 11 37 56 am](https://user-images.githubusercontent.com/704760/30120791-de1b3ca6-92f7-11e7-83c9-f8c3bf6afd5c.png)